README.md

```diff
@@ -83,7 +85,6 @@ This code has been tested in several environments:
 
 - Windows, Linux, MacOS
 - Python 3.6, 3.7, 3.8
```

Comment:
Did 3.8.2 get updated to work with nest_asyncio?

Reply:
Ha! Good catch! So, after looking more carefully at what was going on, I removed `nest_asyncio` totally. In the end, I put in place what I think is a more robust workaround. `nest_asyncio` patches the runtime for python - so it has to be updated all the time. It broke in 3.8.2 because a small error check was added to the python libraries. Now I just run it in a separate thread if there isn't an event loop already running. That seems much more robust.